i18n/src/main/resources/bisq_easy_de.properties

```diff
@@ -279,7 +279,8 @@ bisqEasy.takeOffer.tradeLogMessage={0} hat eine Nachricht gesendet, um das Angeb
 
 bisqEasy.takeOffer.noMediatorAvailable.warning=Es steht kein Mediator zur Verfügung. Sie müssen stattdessen den Support-Chat verwenden.
```

Comment:
`Ã¼` seems you file encoding is not set to UTF8. Also at other following changes.
Btw. it is not needed that devs do the translation (though nice to have). The translation team should pick up (delayed) missing translations. 
Devs usually only set the english versions.

Reply:
that's cool, sure I'll revert this in the next amend